homeassistant/components/homematicip_cloud/__init__.py

```diff
@@ -53,7 +53,7 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
             for entry in hass.config_entries.async_entries(DOMAIN)
         }:
             hass.async_add_job(
```

Comment:
Can we resolve the ignore by using `hass.async_create_task` instead here?

Reply:
yup, done.

The reason I did not do it, is that it is a code change, and some reviewers do not allow code changes unless I have equipment and can test locally.